src/pkg/bundle/remote.go

```diff
@@ -316,3 +318,12 @@ func CheckOCISourcePath(source string) string {
 	}
 	return source
 }
+
+// EnsureOCIPrefix ensures oci prefix is part of provided remote source path, and adds it if it's not
+func EnsureOCIPrefix(source string) string {
+	var ociPrefix = "oci://"
+	if source[:6] != ociPrefix {
```

Comment:
instead of the magic `6`, can we do it by the length of `oci://`?

Reply:
ie. something like ` ociPrefix = len("oci://")` and then `if source [:ociPrefix] ...`

Reply:
unless you prefer creating a new variable, vs using the function result directly... i can update if so

Reply:
nah all good! just wanted to refactor the magic number